README.md

```diff
@@ -654,6 +654,74 @@ instance.delete()
 
 **Do not use delete() in production!**
 
+Bulk-updating sandboxes
+-----------------------
+
+OpenCraft IM provides a tagging mechanism that you can use to update instances in bulk.
+The following script shows how to update multiple instances at once from the Django shell:
+
+```python
+# Author: Brandon DeRosier <bdero@opencraft.com>
```

Comment:
@bradenmacdonald 

> How do you envision that working for things like instance upgrades (e.g. ficus.2 to ficus.3)

The instance points toward a branch called "production", which gets updated from the content of the ficus.2 branch to the ficus.3 branch. A PR could be used to merge it there.

> settings changes (e.g. enable CREATOR_GROUP for all instances)

The settings changes would be listed in the PR, like for our sandboxes currently.

> resource sharding (e.g. moving 1/2 of our instances to a new RabbitMQ server)

That uses configuration settings, no?